sdk/storage/blobs/inc/az_storage_blobs.h

```diff
@@ -53,28 +80,39 @@ typedef struct
   az_span option;
 } az_storage_blobs_blob_upload_options;
 
+/**
+ * @brief Init a client with default options
+ * This is convenient method to create a client with basic settings
+ * Specific options can be modified after initializing the options
+ *
+ * @remark Use this, for instance, when only caring about setting one option by calling this method
+ * and then overriding that specific option
```

Comment:
I am not sure what you mean here.

```suggestion
 * @remark Use this, for instance, when only caring about setting one option by calling this method
 * and then overriding that specific option.
```